std/typecons.d

```diff
@@ -2352,11 +2352,14 @@ struct Nullable(T)
     // that we're assigning to an uninitialized variable.
     else static if (!hasElaborateAssign!T)
     {
-        private T _value = void;
+        private T _value = T.init;
     }
     else
     {
-        static assert(false, "Cannot construct " ~ typeof(this).stringof ~ ": type has no default constructor and overloaded assignment.");
+        static assert(false,
+                      "Cannot construct "~typeof(this).stringof~
```

Comment:
This needs to be properly spaced, e.g.:

`"Cannot construct " ~ typeof(this).stringof ~`